mmv1/products/bigquery/api.yaml

```diff
@@ -248,6 +273,7 @@ objects:
       - specialGroup
       - iamMember
       - view
+      - dataset
```

Comment:
Changing an identity- spooky! That one's been a while. VCR thinks it's safe, so seems fine though.